examples/galaxy_tosca.yaml

```diff
@@ -90,6 +122,15 @@ topology_template:
       properties:
         reference_data: { get_input: reference_data }
         flavor: { get_input: flavor }
+        refdata_repository_name: { get_input: refdata_repository_name }
+        refdata_provider_type: { get_input: refdata_provider_type }
+        refdata_provider: { get_input: refdata_provider }
+        refdata_token: { get_input: refdata_token }
+        refdata_cvmfs_server_url: { get_input: refdata_cvmfs_server_url }
+        refdata_cvmfs_repository_name: { get_input: refdata_cvmfs_repository_name }
```

Comment:
There is another error, there in no input with name: refdata_cvmfs_repository_name.